integrations/service/node/index.ts

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import router from './router';
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: '../../../.env' });
```

Comment:
maybe use `path.join()` to generate the path is a safer way.

I am not sure if `dotenv` will handle the platform-specific separator, otherwise, it will not work on Windows.